ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ccm/disconnected/private/cucushift-installer-rehearse-aws-ipi-ccm-disconnected-private-workflow.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/rehearse/aws/ipi/ccm/disconnected/private/cucushift-installer-rehearse-aws-ipi-ccm-disconnected-private-workflow.yaml",
```

Comment:
I suggest to place it in `aws/ipi/disconnected/private/ccm`

Reply:
ok, thanks! Let me change it.